sandbox/pages/HomePage.vue

```diff
@@ -1,43 +1,157 @@
 <template>
-  <h1>Sandbox Components</h1>
-  <div class="app-layout">
-    <div class="nav-container">
-      <ul>
-        <li>
-          <router-link :to="{ name: 'home'}">
-            Home
-          </router-link>
-        </li>
-        <li>
-          <router-link :to="{ name: 'alert'}">
-            KAlert
-          </router-link>
-        </li>
-        <li>
-          <router-link :to="{ name: 'button'}">
-            KButton
-          </router-link>
-        </li>
-      </ul>
+  <div class="sandbox-header">
+    <router-link
+      class="sandbox-link"
+      :to="{ name: 'home'}"
+    >
+      <h1>
+        <span class="hidden-on-mobile">Kongponents </span>Sandbox
+      </h1>
+    </router-link>
+    <div class="mobile-nav-container">
+      <KToggle v-slot="{ isToggled, toggle }">
```

Comment:
issue: if on mobile, you need to close the slideout if a link in the menu is clicked

Reply:
Done